### ISSUE TITLE
PDUI definition

### ISSUE THREAD

**Issue by gtsitsiridis** [johli/aparent-resnet#2]
Hi,

I couldn't find the code for the computation of the PDUI mentioned in the paper. Would you be so kind as to give a more precise explanation of what the PDUI is exactly and how it was computed? I would really appreciate it!

**Comment by johli** [comment]
Hi, sorry for the slow response.

The PDUI values were estimated using the DaPars algorithm and obtained from other sources/labs.

Specifically, the PDUI values were calculated by Wei Lab and/or Han Lab and originally used (to my knowledge) in the following studies:

Hong, W., et al., 2020. APAatlas: decoding alternative polyadenylation across human tissues. Nucleic acids research, 48(D1), pp.D34-D39. https://doi.org/10.1093/nar/gkz876

Li, L., et al., 2021. An atlas of alternative polyadenylation quantitative trait loci contributing to complex trait and disease heritability. Nature genetics, 53(7), pp.994-1005. https://doi.org/10.1038/s41588-021-00864-5

I obtained the PDUI v7 estimates from this site: https://hanlab.uth.edu/apa/
However, I see now that the site is down.

I obtained the v8 values by emailing Wei Lab.

If you still need the values, feel free to email me through the correspondence address and I'll share them, or alternatively reach out to Wei Lab.

Best,

Johannes
